README.md

```diff
@@ -11,6 +11,8 @@ The format must stay backward compatible in order to be able to validate old eve
 
 Breaking changes to the format must be reflected by a major version update in `_dd.format_version` property.
 
+After editing any schema, you must re-generate the JS + TS definitions ([see below](#build-js-sources--ts-definitions)).
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** Did you consider generating those files only when installing this repository as a dependency?

Reply:
The idea was for this repo to be consumed both as a dependency (web-ui) and as a submodule (browser-sdk) for the time being.
Also, in order to `yarn generate` you need first to `yarn build:deps` to get the _json-schema-to-typescript_ built, and that would happen for everyone developing on web-ui, at least once and every time we update the schemas.

Reply:
indeed